lib/ddtrace/tracer.rb

```diff
@@ -64,8 +64,10 @@ def shutdown!
     #
     # This method makes use of a \ContextProvider that is automatically set during the tracer
     # initialization, or while using a library instrumentation.
-    def call_context(key = nil)
-      @provider.context(key)
+    #
+    # @param thread [Thread] the thread to retrieve the context from. Defaults to current thread.
+    def call_context(thread = nil)
```

Comment:
Minor: "Defaults to current thread" is semantically true, but I guess somewhat confusing seeing as the next line says `thread = nil` . Perhaps this should be

> [Thread] the thread to retrieve the context from. `nil` means current thread. Defaults to `nil`.

...or something similar?